src/platform/user/tests/profile/utilities/index.unit.spec.js

```diff
@@ -117,6 +117,27 @@ describe('Profile utilities', () => {
       );
     });
 
+    it('should handle upstream VET360/VA Profile server errors', () => {
```

Comment:
This functionality was already there, but it didn't have test coverage. I added this to prevent possible regressions.